frontend/mms-Admin/src/app/page.js

```diff
@@ -1,10 +1,13 @@
+"use client";
+import { useGlobalContext } from "../../Context/store";
 import Image from "next/image";
 import { Inter } from "next/font/google";
 import styles from "./page.module.css";
 
 const inter = Inter({ subsets: ["latin"] });
 
 export default function Home() {
+  const { userData, setUserData } = useGlobalContext();
```

Comment:
unused line of code please remove.
and make sure, codecy passes all the checks

Reply:
Okay